src/components/Schedule/Counter.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, useState } from "react";
+import { Icon } from "../Icon";
+import minusIcon from "../../static/svg/minus.svg";
+import plusIcon from "../../static/svg/plus.svg";
+import { Button } from "../Button";
+
+export const Counter = () => {
```

Comment:
at the moment this part looks like 
![image](https://user-images.githubusercontent.com/22029344/58169099-33573980-7c90-11e9-9fdf-d72a72c51628.png)
^ it's not really synced with the design regarding the widths and paddings